packages/router/src/router.js

```diff
@@ -43,7 +48,9 @@ const PrivatePageLoader = ({ useAuth, unauthenticatedRoute, children }) => {
   if (isAuthenticated) {
     return children
   } else {
-    return <Redirect to={unauthenticatedRoute()} />
+    return (
+      <Redirect to={`${unauthenticatedRoute()}?redirectTo=${redirectTo}`} />
+    )
```

Comment:
I'm wondering if we couldn't just grab this from `window.location.pathname`? Then this is just a 1 line change :)

Reply:
I didn't try that  -- while I saw that

`  useAuth = window.__REDWOOD__USE_AUTH,`

accessed window, I saw that `route.props` was used and figured "that path" was part of the `route` just like `route.props.unauthenticatedRedirect` is:

```
              unauthenticatedRoute={
                namedRoutes[route.props.unauthenticatedRedirect]
              }
              redirectTo={route.props.path}
```

I went with that.

But, I imagine `window.location.pathname` would work.

I can try it out.

The the clone doesn't have to worry about that.

Reply:
Awesome, thank you <3

Reply:
@peterp I confirmed that I saw the same redirect behavior with 

```
to={`${unauthenticatedRoute()}?redirectTo=${window.location.pathname}`}
```

so, no need to pass along the redirectTo.

Steps:

1. Logged out
2. Visit private route (e.g.: profile) with unauthenticated route of 'home'
3. Sent back home
4. Login button that also sets the appState for Auth0 

```
appState: { targetUrl: searchParams.get('redirectTo') },
```

5. Login as per usual
6. Now on the /profile and not on the client config callback

Files changes accordingly.